doc/iojs.1

```diff
@@ -533,12 +556,20 @@ If set to 1 then colors will not be used in the REPL.
         type: bool  default: true
   --incremental_marking_steps (do incremental marking steps)
         type: bool  default: true
+  --overapproximate_weak_closure (overapproximate weak closer to reduce atomic pause time)
+        type: bool  default: false
   --concurrent_sweeping (use concurrent sweeping)
         type: bool  default: true
   --trace_incremental_marking (trace progress of the incremental marking)
         type: bool  default: false
   --track_gc_object_stats (track object counts and memory usage)
         type: bool  default: false
+  --track_detached_contexts (track native contexts that are expected to be garbage collected)
+        type: bool  default: true
+  --trace_detached_contexts (trace native contexts that are expected to be garbage collected)
```

Comment:
ooops, this is also v8 document mistake....

https://github.com/v8/v8-git-mirror/blob/05ec0ff08381b3780195069250f658da6040f131/src/flag-definitions.h#L644-L647